bin/run_tests

```diff
@@ -28,4 +28,3 @@ run_docker_test seth_blockhash_test
 run_docker_test seth_blocknum_test
 run_docker_test seth_timestamp_test
 run_docker_test seth_perm_test
-run_docker_test ./rpc/comp_seth_rpc.yaml
```

Comment:
It would be ideal if we could just fix this. If we can't, then we should at least just comment it out and leave an explanation of why. Don't delete this completely because then we will forget about it.

Reply:
It has been added as an acceptance criteria to the story for fixing it

Reply:
that's a good approach.
Here's the Jira Peter updated: https://jira.hyperledger.org/browse/STL-1124

Reply:
I've ipdated the JIRA and am putting a PR together with a fix just now @peterschwarz